### ISSUE TITLE
Add use and symbol elements to the Geometry Properties section

### ISSUE THREAD

**Issue by karip** [w3c/svgwg#525]
The [Geometry Properties](https://svgwg.org/svg2-draft/geometry.html) section doesn't say that the geometry properties apply to use and symbol elements. The sections about [use and symbol elements](https://svgwg.org/svg2-draft/struct.html#SymbolElement) mentions that geometric properties do apply to them.

Make the x and y properties apply to use and symbol elements in the Geometry Properties section and mention those elements in its sizing section. 

Implementors reading the spec may forget that x, y, width and height should be geometry properties for use and symbol elements unless those properties are explicitly mentioned in the Geometry Properties section. As an example, here's a [Chrome bug](https://bugs.chromium.org/p/chromium/issues/detail?id=400725).

**Comment by AmeliaBR** [comment]
Yep, that was definitely an oversight when the decision was made to make use & symbol behave like nested svg as far as geometry properties.

**Comment by dirkschulze** [comment]
@AmeliaBR i think we intentionally did not include symbols (and IIRC use) and added that to the progressive list of things where they potentially apply to in the future. Similar to patterns for instance.

The definition of graphics element does not include the two mentioned elements either. I’d rather go with the current definition in the Geometry Poroperties section but would be fine adding a note that a future spec may include graphical reference elements, symbol or use.

**Comment by dirkschulze** [comment]
Additional note: According to the Chrome issue, Chrome turned all those elements to use presentation attributes. Even mask and pattern where we might need to add more spec text for edge cases. Needs additional testing.
